packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,287 @@
+
+        this.state = {
+            coreGroups: List<Principal>(),
+            coreUsers: List<UserGroup>()
+        };
+    }
+
+    componentDidMount() {
+        getCoreGroups().then((coreGroupsData: List<Principal>) => {
+            this.setState(() => ({
+                coreGroups: coreGroupsData,
+            }));
+        });
+
+        getCoreUsersInGroups().then((coreUsersData: List<UserGroup>) => {
+            this.setState(() => ({
+                coreUsers: coreUsersData,
+            }));
+        });
+    }
+
+    render() {
+        const { model, onSelect } = this.props;
+        const { coreUsers, coreGroups } = this.state;
+        return (
+            <Col xs={12} md={6}>
+                <SectionHeading title="Assignment Options" />
+                <AssignedToGroupInput
+                    model={model}
+                    coreGroups={coreGroups}
+                    onSelect={onSelect}
+                />
+                <DefaultUserAssignmentInput model={model} coreUsers={coreUsers} onSelect={onSelect} />
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+    getIssueNameHelpTip() {
+        return ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP;
+    }
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.singularItemName === null ? 'Issue' : model.singularItemName;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" helpTipBody={this.getIssueNameHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        placeholder="Enter a singular name for this Issue"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class PluralItemNameInput extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+
+    getIssueNameHelpTip() {
+        return ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP;
+    }
+
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.pluralItemName === null ? 'Issues' : model.pluralItemName;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Plural items name" helpTipBody={this.getIssueNameHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="pluralItemName"
+                        type="text"
+                        placeholder="Enter a plural name for this Issue"
```

Comment:
I think you can remove this placeholder prop now that you have a help tip on the label.

Reply:
good point, now removed